pkgs/tools/networking/connman_dmenu/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, connman, dmenu }:
+stdenv.mkDerivation rec {
+  name = "connman_dmenu-${version}";
+  version = "git-29-9-2015";
+
+  src = fetchFromGitHub {
+    owner = "march-linux";
+    repo = "connman_dmenu";
+    rev = "cc89fec40b574b0d234afeb70ea3c94626ca3f5c";
+    sha256 = "061fi83pai4n19l9d7wq6wwj2d7cixwkhkh742c5ibmw1wb274yk";
+    };
```

Comment:
Starting from here your indentation is very inconsistent. Could you please fix that?